Bb style bugs

bbuzzanga opened the pull request:
Addresses a number of open issues
  masking of amplitude (#259)
  DEM download (#265)
  PEP8 style improvements (mask_util, ariaPlot, a bit of ariaExtract and ariaDownload)
 
Addresses a few new issues
  bug in plotting labels
  bug in ariaDownload try/except
 
Print version number (#263)

bbuzzanga committed:
get and show version number

bbuzzanga committed:
bug fix, style upgrade to productPlot

start cleaning productPlot

bug fix, style upgrade

tweak

bbuzzanga committed:
force chmod in ariadownload of netrc

bbuzzanga committed:
remove version info

bbuzzanga committed:
correct amplitude masking

bbuzzanga committed:
small cleanup to extractProduct

bbuzzanga committed:
style improvements to ariaDownload

bbuzzanga committed:
tweak for circleci

bbuzzanga committed:
fix typo

bbuzzanga committed:
fix import

bbuzzanga committed:
fix introduced bug

bbuzzanga committed:
fix introduced bug regarding gdalconst

bbuzzanga committed:
only allow netcdf files

dbekaert commented:
This looks good to me. @sssangha please review.

dbekaert reviewed (approved):
LGTM @bbuzz31 . Thnx

sssangha committed:
chmod fix to address permission issues

Needed to amend chmod with a trailing 0o prefix (i.e. representing octal integers). Otherwise, I kept encountering the following exception:
```
Attempting to obtaining user/pass from .netrc
[Errno 13] Permission denied: '/home/ssangha/.netrc'
```

sssangha committed:
Unfolding long list comprehension

sssangha commented:
@dbekaert @bbuzz31 looks good to me.

I just pushed a couple of changes. Namely I had to pass a patch to `ariaDownload.py` for it to properly set the permissions and read the `.netrc` file on my end. Originally, it was overwriting the permissions and making it unreadable, let alone unwritable. See that separate commit.

I also unfolded a couple of particularly egregious examples of list comprehension in `tsSetup.py`.

That aside, it appears to be in good shape!
